modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-select/src/main/resources/META-INF/resources/select_field.js

```diff
@@ -185,6 +187,12 @@ AUI.add(
 
 						var instance = this;
 
+						var container = instance.get('container');
+
+						var selectGroup = container.one('.form-builder-select-field');
+
+						selectGroup.addClass('active');
+
 						if (!instance.get('readOnly')) {
 							instance.get('container').one('.drop-chosen').toggleClass('hide');
```

Comment:
since we already have the container, can we remove `instance.get('container')` by just `container`?